bower.json

```diff
@@ -22,8 +22,7 @@
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
     "core-selector": "Polymer/core-selector#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "paper-styles": "PolymerElements/paper-styles#0.8.2"
```

Comment:
Why not `^0.8.0`? Are you worried about breaking API changes before we hit `0.9.0`?